docker/Dockerfile.openblas

```diff
@@ -48,6 +50,8 @@ RUN conda create -y -n open3d python=${PYTHON_VERSION} \
 RUN which python \
  && python --version
 
+RUN conda install conda-forge::libgomp
```

Comment:
conda packages can cause conflicts with OS packages and also make reproducibility more difficult, so it's best to avoid them. Can we use the OS libgomp1 here instead (installed on line 31)?

Reply:
yeah I was experimenting, I din't found the reason still, it is a draft PR